docs/module/stp.md

```diff
@@ -9,12 +9,12 @@ Many platforms already support and enable STP by default; this module provides e
 
 The following table describes per-platform support of individual STP features:
 
-| Operating system   | STP | MSTP | RSTP | Per-VLAN (R)STP | Enable per port
-| ------------------ |:---:|:---:|:---:|:---:|:---:|
-| Arista EOS[^EOS]   | ✅  | ✅  | ✅  | ✅ |  ✅ |
-| Cumulus Linux[^CL] | ✅  |  ❌  | ✅  | ❌   |  ✅ |
-| Dell OS10[^OS10]   | ✅  | ✅  | ✅  | ✅ |  ✅ |
-| FRR[^FRR]          | ✅  |  ❌  |  ❌  |  ✅ | ❌   |
+| Operating system   | STP | MSTP | RSTP | Per-VLAN (R)STP | Enable per port | Port type
+| ------------------ |:---:|:---:|:---:|:---:|:---:|:---:|
+| Arista EOS[^EOS]   | ✅  | ✅  | ✅  | ✅ |  ✅ |  ✅ |
+| Cumulus Linux[^CL] | ✅  |  ❌  | ✅  | ❌   |  ✅ |  ✅ |
```

Comment:
It might be worth expanding this to "CL" and "CL NVUE" to match other documentation.

Reply:
done